components/cache/cache_invalidation.rst

```diff
@@ -36,14 +38,14 @@ cache items, as returned by cache adapters::
     $item->tag(array('tag_2', 'tag_3'));
     $cache->save($item);
 
-If ``$cache`` implements :class:`Symfony\\Component\\Cache\\TagAwareAdapterInterface`,
+If ``$cache`` implements :class:`Symfony\\Component\\Cache\\Adapter\\TagAwareAdapterInterface`,
 you can invalidate the cached items by calling
-:method:`Symfony\\Component\\Cache\\TagAwareAdapterInterface::invalidateTags`::
+:method:`Symfony\\Component\\Cache\\Adapter\\TagAwareAdapterInterface::invalidateTags`::
 
     // invalidate all items related to `tag_1` or `tag_3`
     $cache->invalidateTags(array('tag_1', 'tag_3'));
 
-    // if you know the cache key, you can of course delete directly
+    // if you know the cache key, you can of course delete the item directly
```

Comment:
Because terms like "of course", "very easy", "just...", "easily", "simply", etc. are patronizing and most of the times, redundant.